packages/@ember/-internals/glimmer/tests/integration/components/append-test.js

```diff
@@ -283,7 +286,9 @@ class AbstractAppendTest extends RenderingTestCase {
     );
   }
 
-  ['@test appending, updating and destroying a single component'](assert) {
+  [`${testUnless(
+    DEPRECATIONS.DEPRECATE_COMPONENT_TEMPLATE_RESOLVING.isRemoved
+  )} appending, updating and destroying a single component`](assert) {
```

Comment:
Similarly, I think we want to keep this test after deprecation removal?

Reply:
for all of the append tests: https://github.com/emberjs/ember.js/pull/20660#discussion_r1571117787

Reply:
the newly added test covers these use cases (destroy and update)